clients/privacy-center/features/consent/consent.slice.ts

```diff
@@ -67,6 +67,8 @@ export const consentApi = baseApi.injectEndpoints({
           component: ComponentType.PRIVACY_CENTER,
           has_notices: true,
           show_disabled: false,
+          has_config: true,
```

Comment:
I think this became optional at some point, but adding it in to stay consistent with fides-js and to be safer